sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -13,16 +13,21 @@ The Azure Key Vault keys library client supports RSA keys and Elliptic Curve (EC
 
 ## Getting started
 
-### Install the package
+### Include the package
 
-Install the Azure Key Vault keys client library for C++ with [vcpkg](https://github.com/microsoft/vcpkg):
+The easiest way to acquire the C++ SDK is leveraging vcpkg package manager. See the corresponding [Azure SDK for C++ readme section][azsdk_vcpkg_install].
 
-```PowerShell
-# Windows
-vcpkg.exe install azure-security-keyvault-keys-cpp
+To install Azure Security Key Vault Keys package via vcpkg:
 
-# Linux
-./vcpkg install azure-security-keyvault-keys-cpp
+```cmd
+> vcpkg install azure-security-keyvault-keys-cpp
```

Comment:
This won't work as copy/paste.  That's why I was used Windows/Linux commands to support the copy/paste.
But, I won't push hard on it :)

Reply:
I am not pushing hard on this either, but our Getting Started has instructions for both. Here, the instructions would be almost identical (except for `./` prefix), and I think they are pretty understandable to the users of both platforms. But we can update in the future.